app/concepts/todo_list/create.rb

```diff
@@ -0,0 +1,23 @@
+class TodoList::Create < Trailblazer::Operation
+  include Model
+  model TodoList, :create
+
+  contract do
+    property :user
+  end
+
+  def process(params)
+    @model = params[:todo_list_id].nil? ? TodoList.find_or_create_by(name: "Default To-do List") : TodoList.find(params[:todo_list_id])
```

Comment:
This is good enough since you don't know about Trailblazer hooks. But at least it should be factored out into a different method.